CHANGELOG.rst

```diff
@@ -1,5 +1,12 @@
 Changelog
 =========
+[1.4.0] (2022-01-11)
+---------------------
+Deleted
+^^^^^
+- deleted stacapi.py, stacapi_io.py and test_stacapi.py (pystac-client should be used from now on)
+- deleted the StacApi reference in data.py

```

Comment:
please make this a one liner like: 
`removed own stac api client (pystac-client should be used from now on) #159 `